spotbugs/etc/messages.xml

```diff
@@ -7502,6 +7510,21 @@ This format string includes a newline character (\n). In format strings, it is g
 ]]>
      </Details>
   </BugPattern>
+  <BugPattern type="FS_BAD_DATE_FORMAT_FLAG_COMBO">
+    <ShortDescription>Certain SimpleDateFormat format strings may lead to unexpected behavior</ShortDescription>
+    <LongDescription>Certain SimpleDateFormat format strings may lead to unexpected behavior in {1}</LongDescription>
```

Comment:
Instead of the word `Certain` the exact problematic string could be referenced here. It's already added to the BugInstance.

Reply:
fixed